tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+        Float quantile = 0.998
+        Float exonic_fraction = 0.33
+        Float coverage_fraction = 0.05
+        Float min_itd_allele_fraction = 0.07
+        Int max_genomic_breakpoint_distance = 1000000
+        Int min_supporting_reads = 2
+        Int homopolymer_length = 6
+        Int read_through_distance = 10000
+        Int min_anchor_length = 23
+        Int many_spliced_events = 4
+        Int fragment_length = 200
+        Int max_reads = 300
+        Int top_n = 5
+        Int max_itd_length = 100
+        Int min_itd_supporting_reads = 10
+        Int modify_memory_gb = 0
+        Int modify_disk_size_gb = 0
+    }
+
+    Int input_size_gb = ceil(size(bam, "GiB"))
+    Int disk_size_gb = ceil(input_size_gb ) + ceil(size(gtf, "GiB")) + ceil(size(reference_fasta_gz, "GiB")) + modify_disk_size_gb
+    Int memory_gb = ceil(input_size_gb) + modify_memory_gb
+
+    command <<<
+        arriba \
+            -x ~{bam} \
+            ~{if defined(chimeric_sam) then "-c " + chimeric_sam else ""} \
+            -o fusions.tsv \
+            -O fusions.discarded.tsv \
+            -a ~{reference_fasta_gz} \
+            -g ~{gtf} \
+            -G "~{feature_name}" \
+            ~{if defined(exclude_list) then "-b " + exclude_list else ""} \
+            ~{if defined(known_fusions) then "-k " + known_fusions else ""} \
+            ~{if defined(annotate_fusions) then "-t " + annotate_fusions else ""} \
+            ~{if defined(protein_domains) then "-p " + protein_domains else ""} \
+            ~{if defined(wgs_svs) then "-d " + wgs_svs else ""} \
+            -D ~{max_genomic_breakpoint_distance} \
+            -s ~{strandedness} \
+            ~{if length(interesting_contigs) > 0 then "-i " + sep(',', interesting_contigs) else ""} \
+            ~{if length(viral_contigs) > 0 then "-v " + sep(',', viral_contigs) else ""} \
+            ~{if length(disable_filters) > 0 then "-f " + sep(',', disable_filters) else ""} \
+            -E ~{max_e_value} \
+            -S ~{min_supporting_reads} \
+            -m ~{max_mismappers} \
+            -L ~{max_homolog_identity} \
+            -H ~{homopolymer_length} \
+            -R ~{read_through_distance} \
+            -A ~{min_anchor_length} \
+            -M ~{many_spliced_events} \
+            -K ~{max_kmer_content} \
+            -V ~{max_mismatch_pvalue} \
+            -F ~{fragment_length} \
+            -U ~{max_reads} \
+            -Q ~{quantile} \
+            -e ~{exonic_fraction} \
+            -T ~{top_n} \
+            -C ~{coverage_fraction} \
+            -l ~{max_itd_length} \
+            -z ~{min_itd_allele_fraction} \
+            -Z ~{min_itd_supporting_reads} \
+            ~{if mark_duplicates then "" else "-u"} \
+            ~{if report_additional_columns_discard then "-X" else ""} \
+            ~{if fill_gaps then "-I" else ""}
+    >>>
+
+    output {
+        File fusions = "fusions.tsv"
+        File discarded_fusions = "fusions.discarded.tsv"
+    }
```

Comment:
Are these considered intermediate files no one cares about? If not, can we rename them to share a basename with the BAM? This just looks weird to me, unless there's something I'm missing.